### PR TITLE
fix(android): show controls in notification on older androids

### PR DESCRIPTION
**Pull request by KrzysztofMoch**
## Summary
On older android media session is not automatically adding controls to notification (see bellow)
image

This PR address this issue 

### Motivation
Fix issue with missing controls in notification on older androids

### Changes
- refactor notification api to support older devices 

## Test plan
Tested on various emulators with different android version (android 9, 12, 13, 14)

### CODE REVIEW

File: android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackService.kt [hunk 1]
```diff
@@ -148,10 +224,63 @@ class VideoPlaybackService : MediaSessionService() {
...
+
+            handleCommand(commandFromString(actionCommand), session)
+        }
+        return super.onStartCommand(intent, flags, startId)
+    }
+
     companion object {
-        const val COMMAND_SEEK_FORWARD = "SEEK_FORWARD"
-        const val COMMAND_SEEK_BACKWARD = "SEEK_BACKWARD"
+        private const val SEEK_INTERVAL_MS = 10000L
```
Comment (freeboub) [inline android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackService.kt:250]: there no setter to change SEEK_INTERVAL_MS, I think this is creating a regression ...

**Review by freeboub — CHANGES_REQUESTED** [review]
One small comment, else, it looks good to me

File: android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackService.kt (hunk 1 above)
Comment (KrzysztofMoch) [reply android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackService.kt:250]: There is no place when we are changing this (and there wasn't earlier either) - now it's private bcs we dont pass it to `VideoPlaybackCallback` as we handle commands int `VideoPlaybackService`
(not sure if I understands it correctly)

Comment (freeboub) [reply android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackService.kt:250]: OK, sorry, I did a mistake, Confusion with: https://thewidlarzgroup.github.io/react-native-video/component/props#controlsstyles

**Review by freeboub — APPROVED** [review]

**Pull request merged**